NEOSPlus/Logix/Color/Color to CMYK.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using BaseX;
+using FrooxEngine;
+using FrooxEngine.LogiX;
+using FrooxEngine.LogiX.Color;
+
+[Category(new string[] { "LogiX/Color" })]
+[NodeName("Color To CMYK")]
+public class ColorToCMYK : LogixNode
+{
+    public readonly Input<color> Color;
+    public readonly Output<float> C;
+    public readonly Output<float> M;
+    public readonly Output<float> Y;
+    public readonly Output<float> K;
+
+    protected override void OnEvaluate()
+    {
+        color c = Color.Evaluate();
+
+        float rPrime = c.r / 255f;
```

Comment:
Does color store it's values between 0 and 255?

Reply:
im guessing since its RGB conversion which is 0-255

Reply:
The `color` struct stores float values between 0 and 1. This can be found in `BaseX`